docs/Reply.md

```diff
@@ -309,3 +310,11 @@ fastify.get('/teapot', async function (request, reply) => {
 ```
 
 If you want to know more please review [Routes#async-await](https://github.com/fastify/fastify/blob/master/docs/Routes.md#async-await).
+
+<a name="then"></a>
+### .then(fullfilled, rejected)
+
+As the name suggests, `reply.then()` makes `Reply` awaitable.
```

Comment:
I'd refer to some source which explains "await able". We can also use it with promises only.